xml/hardening.xml

```diff
@@ -3075,7 +3075,35 @@ drwxrwxrwt 18 root root 16384 Dec 23 22:20 /tmp</screen>
      To locate files not owned by any user or group, use the following command:
     </para>
 
-<screen>&prompt.root;find / -path /proc -prune -o -nouser -o -nogroup</screen>
+<screen>&prompt.root;find /bin /boot /etc /home /lib /lib64 /opt /root /sbin /srv /tmp /usr /var -nouser -o -nogroup</screen>
+
+    <para>
+    You might need to extend the list of directories that are searched if you have a different filesystem structure.
+    </para>
+
+    <para>
+     A different problem are files that were not installed via the packaging
+     system and therefor don't receive updates. You can check for such files with
+     the following command:
+    </para>
+
+<screen># find /bin /lib /lib64 /usr -path /usr/local -prune -o -type f -a -exec /bin/sh -c "rpm -qf {} &amp;> /dev/null || echo {}" \;</screen>
+
+    <para>
+     You should run this command as an untrusted user (e.g. nobody) since crafted filenames might lead to command
```

Comment:
Remove "You should". "Run this command ..." is fine. "e.g." -> "for example"

Reply:
Thanks